api_guard/dist/types/index.d.ts

```diff
@@ -249,7 +249,7 @@ export interface ObjectUnsubscribedError extends Error {
 
 export declare const ObjectUnsubscribedError: ObjectUnsubscribedErrorCtor;
 
-export declare const observable: string | symbol;
+export declare const observable: string | typeof Symbol.observable;
 
```

Comment:
Why has this changed? The last time this happened, IIRC, it was in that PR that sought to bump to TS 4.3. Are you sure you have 4.2 installed?